src/main/java/lotto/Prize.java

```diff
@@ -0,0 +1,43 @@
+package lotto;
+
+import java.util.Arrays;
+
+public enum Prize {
+    ZERO(0, 0),
+    FIRST(1, 0),
+    SECOND(2, 0),
+    THIRD(3, 5000),
+    FOURTH(4, 50000),
+    FIFTH(5, 1500000),
+    SIXTH(6, 2000000000);
+
+    private final int rank;
+    private final int price;
+
+    Prize(int rank, int price) {
+        this.rank = rank;
+        this.price = price;
+    }
+
+    public static Prize prizeByMatchedCount(int matchedCount) {
+        return Arrays.stream(Prize.values())
+            .filter(prizes -> Integer.compare(prizes.rank, matchedCount) == 0)
```

Comment:
같은지 비교하는 식은 아래와 같이 작성하는 것이 가독성이 더 좋은 것 같기도 한데, 어떻게 생각하시나요?
```suggestion
            .filter(prizes -> prizes.rank == matchedCount)
```
`filter` 에서 받는 인자의 이름도 `Prize` 객체 하나이므로 `prize` 라는 단수형 이름을 쓰면 어떨까요?